WorkflowUI/Tests/ViewControllerDescriptionTests.swift

```diff
@@ -148,4 +148,23 @@
         }
     }
 
+    class ViewControllerDescription_KindIdentifierTests: XCTestCase {
+        private final class VC1: UIViewController {}
+        private final class VC2: UIViewController {}
+
+        func test_kind() {
```

Comment:
what's the expected behavior if you had, say, `class VC3: VC2`?

Reply:
Same as before; whatever `is` returns for the given types.